src/focus.service.ts

```diff
@@ -554,6 +554,13 @@ export class FocusService {
       return false;
     }
 
+    for (let parent = el; parent; parent = parent.parentElement) {
+      const record2 = this.registry.find(parent);
```

Comment:
You can name this `record` instead of `record2`. const/let declarations are block-scoped.